client/app/pages/queries-list/QueriesList.jsx

```diff
@@ -0,0 +1,247 @@
+import { extend } from 'lodash';
+import moment from 'moment';
+import React from 'react';
+import { react2angular } from 'react2angular';
+import { FavoritesControl } from '@/components/FavoritesControl';
+import { QueryTagsControl } from '@/components/tags-control/QueryTagsControl';
+import { SchedulePhrase } from '@/components/queries/SchedulePhrase';
+import { BigMessage } from '@/components/BigMessage';
+import { NoTaggedObjectsFound } from '@/components/NoTaggedObjectsFound';
+import { EmptyState } from '@/components/empty-state/EmptyState';
+import ItemsList from '@/pages/ItemsList';
+import { $location, $rootScope } from '@/services/ng';
+import { Query } from '@/services/query';
+import { currentUser } from '@/services/auth';
+import { durationHumanize } from '@/filters';
+import { formatDateTime } from '@/filters/datetime';
+import './queries-list.css';
+
+class QueriesList extends ItemsList {
+  static sidebarMenu = [
+    {
```

Comment:
I think we should be able to use this list to hook up the routing as well, similar how I did this in #2888?

Reply:
Yes, I'll revisit it after #2888 will be merged :+1: